File: gc/go/main1.go
package main

import (
	"fmt"
	"time"
)

type Data struct {
	SIZE int
	arr  []int
}

func main() {
	fmt.Println("Hello")
	start := time.Now()
	var checksum int64 = 0
	for i := 0; i < 1000000; i++ {
		data := new(Data)
		arr := make([]int, 1024)
		data.arr = arr
		data.SIZE = 1024
		for j := 0; j < data.SIZE; j++ {
			data.arr[j] = j
			checksum += int64(data.arr[j])
		}
	}
	end := time.Now()
	fmt.Println(checksum)
	fmt.Printf("time (ms): %d\n", end.Sub(start)/1000000)
}

File: gc/java/Main1.java
public class Main1 {
	
	public static void main(String[] args) {
		System.out.println("Hello");
		long start = System.currentTimeMillis();
        long checksum = 0;
		for (int i = 0; i < 1_000_000; i++) {
			Data data = new Data();
			for (int j = 0; j < Data.N; j++) {
				data.arr[j] = j;
                checksum += data.arr[j];
			}
		}
		long end = System.currentTimeMillis();
        System.out.println(checksum);
		System.out.format("time (ms): %d\n", end - start);
	}

}

class Data {
	static final int N = 1024;
	int[] arr = new int[Data.N];
}